connection.php

```diff
@@ -0,0 +1,11 @@
+
```

Comment:
rm this file, to keep it consistent, write your database connection into your profilepage.php file